<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DecryptionDialog</class>
 <widget class="QDialog" name="DecryptionDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>298</width>
    <height>224</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Note decryption</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog#NoteDecryptionDialog {
margin: 4px;
spacing: 4px;
}</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="infoTextLabel">
     <property name="text">
      <string>Enter passphrase to decrypt content:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="passwordLineEdit">
     <property name="text">
      <string notr="true"/>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="showPasswordCheckBox">
     <property name="text">
      <string>Show password</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="onErrorTextLabel">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="hintLabel">
     <property name="text">
      <string>Hint:</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="rememberPasswordCheckBox">
     <property name="text">
      <string>Remember passphrase during the session</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="decryptPermanentlyCheckBox">
     <property name="text">
      <string>Decrypt permanently</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DecryptionDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DecryptionDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
